labs/authentication-passport/README.md

```diff
@@ -0,0 +1,185 @@
+# Passport Strategy Adapter
+
+_Important: We suggest users understand LoopBack's authentication system[Link
```

Comment:
We strongly recommend that users ...

Reply:
And I updated the link as well since https://github.com/strongloop/loopback-next/pull/2977 get merged